### ISSUE TITLE
Suggestions for the documentation

### ISSUE THREAD

**Issue by Stefan-Heimersheim** [EGaraldi/corecon#6]
Hello,

First let me say that I think this project will be very useful and the collection of constraints is a great idea. I have been taking a look at it for the [JOSS Review](https://github.com/openjournals/joss-reviews/issues/2495) and want to suggest a few improvements to the documentation here. (I'll put unrelated comments or bug reports in separate issues to keep everything clear.)

The table of contents on the [readthedocs](https://corecon.readthedocs.io/en/latest/index.html) page doesn't contain all the items (e.g. tutorial, the constraints, function docs) and when you click on one of the links (examples from before) the table of contents disappears. Would it be possible to include them all in the sidebar? That would make it considerably easier to navigate and find things? See e.g. [cobaya](https://cobaya.readthedocs.io/en/latest/input.html) or [this repo](https://gamma-astro-data-formats.readthedocs.io/en/latest/general/coordinates.html) for examples.

You have some nice plots illustrating the various constraints (e.g. [here](https://corecon.readthedocs.io/en/latest/datarst/ionized_fraction.html)). Could you add/link the source code for those plots as well? This would be helpful for contributors adding their constraints so they can update the plots, but also for users using the constraints.

Best,
Stefan

**Comment by Stefan-Heimersheim** [comment]
Another small point: Would you mind adding a note to your docs on how to report issues or problems with the software and to seek support, fulfilling the requirements for the [community guidelines](https://joss.readthedocs.io/en/latest/review_criteria.html#community-guidelines)?

**Comment by EGaraldi** [comment]
Hi,
I have restructured the docs a bit so now everything is linked. I may do a deeper restructuring/improvement later, when other things are fixed/changed. I have also included a few lines on how to report issues. Concerning the plot code, it's a messy and automated script, which I am afraid may create more confusion than bring clarity. I'd rather stick with what is currently in the tutorial, which is essentially a cleaned-up version of the automatic script.
